.circleci/config.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:current
```

Comment:
Do we really want "current" for version? There could be breaking changes in a future Node, for a future build here causing random failures, and also then rebuild on any historic build could fail.